entry_test.go

```diff
@@ -22,7 +22,7 @@ func TestEntryWithError(t *testing.T) {
 
 	logger := New()
 	logger.Out = &bytes.Buffer{}
-	entry := NewEntry(logger)
+	entry := NewEntry(logger, 1)
```

Comment:
what I'm saying is that the test assertions need to check that the right `callrer: src.go:42` is printed